runelite-client/src/main/java/net/runelite/client/game/ItemManager.java

```diff
@@ -331,6 +319,25 @@ public int getItemPrice(int itemID)
 		return price;
 	}
 
+	public int getItemPrice(int itemID, int quantity)
+	{
+		int price = 0;
+
+		if (itemID == ItemID.MASTER_SCROLL_BOOK)
+		{
+			final int totalEmptyBookPrice = getItemPrice(MASTER_SCROLL_BOOK_EMPTY) * quantity;
+			price = totalEmptyBookPrice;
+
+			for (Map.Entry<Varbits, Integer> entry : VARBIT_MASTERBOOK_ITEMIDMAP.entrySet())
+			{
+				final int amount = client.getVar(entry.getKey());
+				price += getItemPrice(entry.getValue()) * amount;
+			}
+		}
+
+		return price;
```

Comment:
so this works only for master scroll book? you should also make it work for normal items.